src/z3c/form/browser/radio.py

```diff
@@ -44,14 +44,15 @@ def renderForValue(self, value):
         terms = list(self.terms)
         try:
             term = self.terms.getTermByToken(value)
+            id = '%s-%i' % (self.id, terms.index(term))
```

Comment:
I'd think this line should be in an `else` branch of the `try-except` to make clear that it is not the possible source of the `LookupErrror`.